ui/dist/index.html

```diff
@@ -0,0 +1,11 @@
+<! DOCTYPE html>
```

Comment:
There are few files in the `ui/dist/` folder, is that on purpose?

Reply:
Yes, this is the UI build and published. I haven't investigated enough but its the easy way to have GitHub Pages to display something. We need the builded file because how GitHub will display the correct website?